insights/parsers/ceph_version.py

```diff
@@ -54,7 +54,9 @@
     "12.2.4-42": {'version': "3.0.5", 'major': '3', 'minor': '0', 'downstream_release': '5'},
     "12.2.5-42": {'version': "3.1", 'major': '3', 'minor': '1', 'downstream_release': '0'},
     "12.2.5-59": {'version': "3.1.1", 'major': '3', 'minor': '1', 'downstream_release': '1'},
-    "12.2.8-52": {'version': "3.2", 'major': '3', 'minor': '2', 'downstream_release': '0'}
+    "12.2.8-52": {'version': "3.2", 'major': '3', 'minor': '2', 'downstream_release': '0'},
+    "12.2.8-89": {'version': "3.2.1", 'major': '3', 'minor': '2', 'downstream_release': '1'}
+    
```

Comment:
Hi @gauravsitlani, the Jenkins complaints the following messages from `flake8` check
~~~
./insights/parsers/ceph_version.py:59:1: W293 blank line contains whitespace
~~~

Would you please to remove this blank?
Btw, you can run the `flake8` command in your local as well.

Reply:
sure , I'll remove this blank.

Reply:
@xiangce thanks for sharing, removed the blank space.